Formula/php.rb

```diff
@@ -86,6 +92,14 @@ def install
     # Prevent homebrew from harcoding path to sed shim in phpize script
     ENV["lt_cv_path_SED"] = "sed"
 
+    # Each extension that is built on Mojave needs a direct reference to the
+    # sdk path
+    if MacOS.version == :mojave
```

Comment:
Are we sure we want `MacOS.version == :mojave` throughout and not `if MacOS.sdk_path_if_needed` or `if DevelopmentTools.clang_build_version >= 1000`?

Reply:
I guess `MacOS.sdk_path_if_needed` would be well suited I'll swap